docs/conf.py

```diff
@@ -63,7 +63,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
was seeing a warning building the docs that no language was set so set it to English